testing/smoke/basic-upgrade/test.sh

```diff
@@ -16,13 +16,7 @@ fi
 MAJOR_VERSION=$(echo ${VERSION} | cut -d '.' -f1 )
 MINOR_VERSION=$(echo ${VERSION} | cut -d '.' -f2 )
 
-if [[ ${MAJOR_VERSION} -eq 7 ]]; then
-    ASSERT_EVENTS_FUNC=legacy_assertions
-    INTEGRATIONS_SERVER=false
-    get_latest_patch "${MAJOR_VERSION}.${MINOR_VERSION}"
-    LATEST_VERSION=${MAJOR_VERSION}.${MINOR_VERSION}.${LATEST_PATCH}
-    PREV_LATEST_VERSION=$(echo ${MAJOR_VERSION}.${MINOR_VERSION}.$(( ${LATEST_PATCH} -1 )))
```

Comment:
Upgrading from `7.17` to `8.x` is still supported until `9.0` is released. Why did you decide to already remove the upgrade tests for `7.17 -> 8.x`?